### ISSUE TITLE
Simplify the triangle icon that let people go to the voting triangle so they do not think it is meaningful

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-client#90]
Every time my father wanted to open the voting triangle he would click on the small voting triangle in the question page next to a proposal, which would open the big triangle to vote. This is fine, but he would also try to click on the part of the triangle that represented what he wanted to say. On the green side if he agreed. On the red side if he disagreed. And so on. As those area are tiny it took him considerable time, for something that is pointless since the system was not storing this information at all. Only the vote on the big triangle counts. 
![screen shot 2014-12-21 at 17 55 12](https://cloud.githubusercontent.com/assets/1333674/5518896/1355ed46-893a-11e4-81ee-2a53e23aada4.png)

Maybe simplifying this triangle so that it is not even divided. Maybe a smooth triangle would be better. With a message popping up: vote. Or just the words: "evaluate this proposal".

![screen shot 2014-12-21 at 17 59 11](https://cloud.githubusercontent.com/assets/1333674/5518903/9f90191c-893a-11e4-82e1-57109baaa647.png)

**Comment by pietrosperoni** [comment]
Hi Derek, no the popup window is surely not enough. Very few people reads the popup window.

**Comment by pietrosperoni** [comment]
Ok, I see this seem to be confusing. Please let's just change the icon into something which has smoother colors (the icon above would be great), or just the phrase VOTE

**Comment by giovanisp** [comment]
@derekpa use a triangle icon like the ones used for comments and questions, same size. Maybe a different color when user has already voted that proposal. It makes Us consistent.


-------- Messaggio originale --------
Da: Pietro Speroni <notifications@github.com> 
Data: 02/01/2015  11:37  (GMT+01:00) 
A: fairdemocracy/vilfredo-client <vilfredo-client@noreply.github.com> 
Cc: Giovani Spagnolo <giovani@webyes.com.br> 
Oggetto: Re: [vilfredo-client] Simplify the triangle icon that let people go
  to the voting triangle so they do not think it is meaningful (#90) 

Ok, I see this seem to be confusing. Please let's just change the icon into something which has smoother colors (the icon above would be great), or just the phrase VOTE

**Comment by derekpa** [comment]
Fine, I'll change it.

Did you understand what I said though? That your father already expected it to work that way because that's how it worked before, which does not imply a new user would expect the same thing?

**Comment by pietrosperoni** [comment]
Maybe, but it has also to work for old users

**Comment by giovanisp** [comment]
Guys, I propose this "standard" user interface:

![image](https://cloud.githubusercontent.com/assets/180794/5736812/9bd8460c-9bda-11e4-91f0-7f6299d9e1b2.png)

**Comment by pietrosperoni** [comment]
@giovanisp The color must be continuous, as the voting is not discreet. It is Vilfredo that will decide if a certain vote counts as a +1 or a -1.
For the rest I agree.

**Comment by giovanisp** [comment]
I meant the colors indicate if the user voted on the "agree"/"disagree" or "don't understand" area of the triangle at that moment (we said that for now the point is fixed, right?) .  Colors could change when we have the dynamic triangle.

**Comment by derekpa** [comment]
The proposal box is already coloured depending on how they voted or if they voted.

**Comment by giovanisp** [comment]
yeah, we can apply the same color for now, so we don't have different color schemes.

**Comment by pietrosperoni** [comment]
Yep, the same color as the proposal box. And a different color which cannot be obtained inside the triangle, if you did not vote :-)

**Comment by giovanisp** [comment]
the blue color, like the other icons ;-)

**Comment by pietrosperoni** [comment]
if it doesn't get confused with voting at the lower part of I don't understand. Although in case it would make sense... :-)

**Comment by derekpa** [comment]
Let's move this to 2.1
